app.py

```diff
@@ -41,6 +38,14 @@ def print_output(deliveries):
         print_delivery(delivery)
 
 def run(T2, T3, T4, pizzas):
+    deliveries = []
+    #create deliveries (tuple with id's) and add them to deliveries
```

Comment:
We might want to return Pizza instances, so we can score them with their ingredients